index.html

```diff
@@ -191,30 +176,56 @@ <h2>Multi-Post Stories </h2>
                         <li class="lang"> Ruby</li>
                     </ul>
                     <div>
-                        <button class="button" type="button" onclick="">See Project</button>
+                        <button id="modalBtn4" class="button" type="button" onclick="true">See Project</button>
                     </div>
                 </div>
             </div>
         </section>
+        <!-- Popup Window -->
+        <div id="simpleModal" class="modal">
+            <div class="modal-content">
+                <div class="header">
+                    <span id="closeBtn">&times;</span>
+                <h2>Multi-post story</h2>
+                </div>
+                <img class="imagine" src="./Images/snapshot.png" alt="">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+                    and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+                    typesetting ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.
+                </p>
+                <div class="content">
+                    <p>| html</p>
+                <p>| Ruby on rails |</p>
+                <p>css |</p>
+                </div>
+                <div class="modal-button">
+                    <button class="btn" type="submit">See Live<img src="./Images/arrow.png" alt="image"></button>
+                    <button class="btn btn2" type="submit">See Source<img src="./Images/github1.png" alt="image"></button>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] Your modal pop-up is hardcoded. The expected behaviour is that when the "See Project" button is clicked, a pop-up modal with more descriptions about that particular project shows up. Kindly implement this with JavaScript 👍 
- [ ] On mobile view, kindly reduce the gap between the technologies and the buttons as shown in the image below 👇 
### Figma Mobile Design
![image](https://user-images.githubusercontent.com/46839548/190877923-0ab15733-07ee-4b27-872d-1660a1641bcd.png)
### Your Implementation
![image](https://user-images.githubusercontent.com/46839548/190877233-404939e5-40b6-4f2b-b966-88211a4a9481.png)

- [ ] On desktop view, kindly try your best to follow the design guidelines as closely as possible.
### Figma Desktop Design
![image](https://user-images.githubusercontent.com/46839548/190877843-b9df8e0b-d7f0-4340-a288-cc17c8cc5d82.png)
### Your Implementation
![image](https://user-images.githubusercontent.com/46839548/190878118-59480095-07f7-49b2-be7a-200741d1d7c1.png)